zimui/src/components/channel/tabs/VideosTab.vue

```diff
@@ -11,6 +11,7 @@ import type { Playlist } from '@/types/Playlists'
 const main = useMainStore()
 const videos = ref<VideoPreview[]>([])
 const playlist = ref<Playlist>()
+const isLoading = computed(() => videos.value.length === 0)
```

Comment:
Same remark about empty videos list

Reply:
Changed it to a ref in [d1930a0](https://github.com/openzim/youtube/pull/289/commits/d1930a05040e8a51b581c2be289d366f87a93806)